app/utils/AtedUtils.scala

```diff
@@ -150,4 +150,6 @@ object AtedUtils {
     request.body.asFormUrlEncoded.map(c => c ++ params)
   }
 
+  def getEarliestDate(date1: LocalDate, date2: LocalDate) = if (date1.isBefore(date2)) date1 else date2
```

Comment:
I think this function is simple enough to move in to the place it's called

Reply:
A function like this should be in the date library (in my opinion) and I see ATEDutils file as the next best thing to that. It keeps all the code in the controller - controller specific. Also it is there if a future developer needs it.